controller/refugee_controller.py

```diff
@@ -1,10 +1,19 @@
 ###---- Manage Refugee Menu ----
 from datetime import date
+from enum import Enum
+from typing import Type
 
 from models.camp import Camp
 from models.refugee import Refugee
 
 
+def list_medical_condition_types() -> Type[Enum]:
+    """
+    List all options for medical condition type.
+    """
+    pass
```

Comment:
Currently returning nothing. Try
```
return Refugee.MedicalCondition
```

Reply:
Sorry for the misleading. Refugee_controller is not included in this PR.